### DISCUSSION TITLE
Web

### DISCUSSION THREAD

**Discussion by Anatolysco** [4tals/LinksForIsrael#523] [Ideas]
**I want to start a discussion about a brand-new design for our web version!**

In general, the main idea is to use the components of the mobile version. I added an interactive gallery, simplified navigation (no more list of categories and panels on the right - everything is organized as tags, similar to the mobile. As in mobile, we're talking about a smooth content organization that give an ability to find what you're looking for with just a few taps.

So, let's discuss about and bring this new version to life along with the mobile.

[Desktop  demo (hot spots will lead your way):](https://www.figma.com/proto/BxHCYKopVxDdMmuIoOQ0AL/Look-and-Feel?type=design&node-id=786-2800&t=jVm3V5lPTF227135-0&scaling=scale-down-width&page-id=0%3A1&starting-point-node-id=786%3A2800&hide-ui=1)

[Figma (with a design mock ups)](https://www.figma.com/file/BxHCYKopVxDdMmuIoOQ0AL/Look-and-Feel?type=design&node-id=663-20847&mode=design&t=get3Vt6PdcAEDklX-4)

Looking forward to diving into this with you all! @4tal @yanai101 @ohadschn @oriharel
Adds to the discussion those you consider necessary and feel free to ask questions

![Desktop](https://github.com/4tals/LinksForIsrael/assets/12582720/14909833-3eb4-43cb-b380-38e6dd8c691a)

**Comment by 4tal** [comment]
1. באיטרציית עיצוב הראשונה אפשר להוריד את אלו:
<img width="330" alt="image" src="https://github.com/4tals/LinksForIsrael/assets/6404507/79e1018d-ad39-451a-b78b-239f2413e278">
<img width="946" alt="image" src="https://github.com/4tals/LinksForIsrael/assets/6404507/7fb645fe-aecf-4cba-a348-bae0224362fc">


2. חסר כמות יוזמות לפי קטגוריה:
<img width="938" alt="image" src="https://github.com/4tals/LinksForIsrael/assets/6404507/268aa99e-7a6a-4012-b797-218027533864">
3. כיום כל הלינקים בכרטיסייה כבר מובילים ללינק הרלוונטי. כדאי להפוך את הלינקים לפעילים רק כשהכרטיסייה פתוחה? ולא לאפשר מעבר ללינקים ישירות מהמסך הראשי?




ש

**Reply by Anatolysco** [reply]
בנוסף, אולי ( לא חובה) גם אין צורך בpop up בלחיצה על כרטיסיה כי זה כפל: יש איקונים ספציפיים שמובילים ישר וגם אייקון של לינק עצמו

**Reply by 4tal** [reply]
הלינקים קטנים, לא הכי נוח. נראה לי הכי נוח אם לחיצה על כל מקום בכרטיסיה פותחת פופאפ עם שאר הלינקים קצת יותר גדולים. הרבה יותר נוח להשתמש

**Reply by Anatolysco** [reply]
הורדתי לינקים עליונים

**Comment by Anatolysco** [comment]
As per last meeting with @4tal there was an agreement for some changes (web wise). Please, go over the updated web version:
Simpler navigation, rid off the pop ups  are destructing smooth user experience - from now on the full topic opens to the side, without interfering with viewing the main content (like in Google).

My approach here is for a search engine - clean, clear flow in order to find information, ease of use:
@4tal / @omer-friedman / @yanai101 / @ohadschn 

Please review and leave comment (here or [Figma](https://www.figma.com/file/BxHCYKopVxDdMmuIoOQ0AL/UX-%2F-UI?type=design&node-id=250-28763&mode=design&t=pAerouhNQCvQrvOR-4))

Demo (just use the hot spots / click what's clickable)
https://www.figma.com/proto/BxHCYKopVxDdMmuIoOQ0AL/UX-%2F-UI?type=design&node-id=786-2800&t=M27CNmQDfBgVrLNV-0&scaling=scale-down-width&page-id=0%3A1&starting-point-node-id=786%3A2800&hide-ui=1
